Documentation/DocuBlocks/Rest/Cursors/post_api_cursor.md

```diff
@@ -6,6 +6,23 @@
 
 A JSON object describing the query and query parameters.
 
+@RESTHEADERPARAM{x-arango-allow-dirty-read,string,optional}
+If the "x-arango-allow-dirty-read" header is given, then it must contain
+the value `true`. This will allow the coordinator to ask any shard
+replica for the data and not just the leader. However, this may then
+produce "dirty reads", see
+[Dirty Reads](./document-address-and-etag.html#dirty-reads)
+for details. Note that if this operation is part of a read-only streaming
+transaction (see "x-arango-trx-id" header below), then it is the
+transaction creation operation that decides about dirty reads, and not
+this header for the individual read operation.
+
+@RESTHEADERPARAM{x-arango-trx-id,string,optional}
```

Comment:
Should the type be `integer`? Seems to be a `uint64` internally. Everything is a string over HTTP initially, so it's not wrong, but I believe that the Swagger type should indicate the internal type, or rather what a client should map it to.

Reply:
I added this part of the documentation in this PR, since it was obviously forgotten. I think the user facing side of the API only has an integer in there. However, cluster-internally, this header can be more complicated and thus is a string.

Reply:
Thanks for the explanation. Looks like it's in fact a string to users:
`{"code":201,"error":false,"result":{"id":"2199063338824","status":"running"}}`
`typeof trx.id() // string`